packages/tldraw/src/lib/ui/components/primitives/TldrawUiInput.tsx

```diff
@@ -176,6 +179,8 @@ export const TldrawUiInput = React.forwardRef<HTMLInputElement, TLUiInputProps>(
 					onChange={handleChange}
 					onFocus={handleFocus}
 					onBlur={handleBlur}
+					onCompositionStart={() => (isCompositioning.current = true)}
```

Comment:
let's create some handlers above using `useCallback` to avoid any possible re-renders